src/common/exceptions/unprocessable-entity.exception.ts

```diff
@@ -5,16 +5,39 @@ import { UnprocessableEntityError } from '../interfaces';
 export class UnprocessableEntityException extends Error {
   readonly status: number = 422;
   readonly name: string = 'UnprocessableEntityException';
-  readonly message: string;
+  readonly message: string = 'Unprocessable entity';
+  readonly code?: string;
 
-  constructor(errors: UnprocessableEntityError[], readonly requestID: string) {
+  constructor(
+    {
+      code,
+      errors,
+      message,
+    }: {
+      code?: string;
+      errors?: UnprocessableEntityError[];
+      message?: string;
+    },
+    readonly requestID: string,
```

Comment:
why not roll requestId into the constructor object as well?

Reply:
I don't think there is a way to use the `readonly` property + constructor argument shorthand that way unfortunately.

Reply:
Updated by adding it to the object and just manually assigning property in the constructor.